lib/init/grass.py

```diff
@@ -2030,6 +2030,12 @@ def parse_cmdline(argv, default_gui) -> Parameters:
 
     Returns Parameters object used throughout the script.
     """
+    # For the subcommands, we keep a list here which allows use not to import
```

Comment:
```suggestion
    # For the subcommands, we keep a list here which allows us not to import
```

This is a great step especially given GDAL has recently added subcommands.